beacon-chain/core/transition/transition.go

```diff
@@ -147,25 +147,15 @@ func ProcessSlotsUsingNextSlotCache(
 	ctx, span := trace.StartSpan(ctx, "core.state.ProcessSlotsUsingNextSlotCache")
 	defer span.End()
 
-	// Check whether the parent state has been advanced by 1 slot in next slot cache.
```

Comment:
This is a very simple function that was overly commented making it less readable. I removed some useless variables and most comments.